libs/sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -632,6 +633,11 @@ namedProperties
     : ident withProperties?
     ;
 
+copyFromOptions
+    : (RETURN SUMMARY)? (FAIL FAST)?
```

Comment:
Could we add an option for withProperties instead of going with new keywords?

Reply:
Thanks Jordi, made the change.